hmcts/pre-api
# S28 3192: Default Endpoint for Live Playback

### JIRA ticket(s)
- S28-3192


### Change description
- All live playbacks now use the same streaming endpoint (similar to VOD playback).
- When `default-live` streaming endpoint does not exist, create and start the streaming endpoint (this should only ever happen once per environment). 




> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**
- On MediaKind, delete the `default-live` streaming endpoint if it exists. Then:
    - Create and start live event, start CVP recording, call the play live event endpoint. See endpoint url contains the string `default-live` (it will be something like `https://ep-default-live-pre-mediakind-stg.uksouth.streaming.mediakind.com/...`)
    - See live output url works
    - Stop live event- see streaming endpoint remains running and is not deleted
    - Create + start another live event, start CVP recording + call the play live event endpoint again. See it uses the same streaming endpoint + there is not a new streaming endpoint created in mediakind (since it is reusing the `default-live` one)
    - See live output url works
    - Stop live event- see streaming endpoint remains running and is not deleted
- You should notice that the first time the calling the streaming endpoint will take a lot longer to setup but after that it will be relatively quick for subsequent calls. (you will still get the ~20 second wait for the locator but not the 1-7 mins for the endpoint).
